torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -187,9 +193,22 @@ struct GraphFuser {
   }
 
   bool isFusable(Node* node) {
+    if (callback_) {
```

Comment:
It is weird to only partially use isFusableCallback. The default behavior in the graph fuser should itself be turned into a callback and then we should use that callback for prim::FusionGroup creation.